lib/remarkable_ruby/object.rb

```diff
@@ -0,0 +1,29 @@
+module RemarkableRuby
+  class Object
```

Comment:
Object might be a little too general of a name for this class. rmcl calls it Item. Here are some other suggestions for an entity that can be either a file or a folder: https://stackoverflow.com/questions/16197641/what-is-a-name-that-can-represent-both-a-file-or-directory

some options:
* FileSystemNode
* FileSystemEntry
* FSEntry
* RMNode / RMEntry etc

have any preferences?

Reply:
I kinda like Item. It's short and I think much better than Object. Maybe we should change the `client.documents` to something like `client.items` or `client.list_items`?